test/cljs/rems/test_util.cljs

```diff
@@ -22,7 +25,8 @@
 (deftest test-linkify
   (let [link [:a {:target :_blank :href "http://www.abc.com"} "http://www.abc.com"]]
     (testing "retain original string"
-      (is (= (apply str (linkify "a b c") "a b c"))))
+      (is (= (linkify "a b c")
+             ["a b c"])))
```

Comment:
Is the expected and actual value reversed in this test? I guess they should be in reverse order?

Reply:
True, even though other tests in same file specify expected/actual as elsewhere. Fixed